app/templates/base.html

```diff
@@ -47,6 +47,21 @@ <h2>Yalies</h2>
             {% endblock %}
         </main>
         <footer>
+            <!--Scroll-to-top button (source:https://www.scrolltotop.com/)-->
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
We can't add a dependency on jQuery just for this button. Try this StackOverflow thread for how to do it in vanilla JS in just a few lines: https://stackoverflow.com/questions/15935318/smooth-scroll-to-top